omod/src/main/java/org/openmrs/module/spa/SpaDirectoryResolver.java

```diff
@@ -84,6 +84,8 @@ private static void resolveDirectory(String spaDirectory) {
                 SpaDirectoryResolver.spaDirectory.set(Paths.get(OpenmrsUtil.getApplicationDataDirectory(), DEFAULT_FRONTEND_DIRECTORY).normalize().toAbsolutePath().toString());
             }
             SpaDirectoryResolver.spaDirectory.set(spaDirectoryPath.normalize().toString());
+        } else {
+            SpaDirectoryResolver.spaDirectory.set(spaDirectory);
```

Comment:
Good catch! Ugh... maybe this explains the issue KenyaEMR was having? cc: @ojwanganto Basically if you had a GP that set the frontend directory to a different directory and that directory was expressed as an absolute path, it would've just returned nothing.